packages/salesforcedx-utils-vscode/src/commands/index.ts

```diff
@@ -5,9 +5,6 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-import { ChannelService } from './channelService';
-export const channelService = ChannelService.getInstance();
```

Comment:
Exposing the channel service like this prevents us from initializing the service with a specific name. Instead, this gets initialized at startup as a channel with the default name.